docs/general/faq.md

```diff
@@ -6,14 +6,14 @@ LWCC extends the power of Chart.js and supports all the languages, included righ
 ## Where can I use LWCCBuilder?
 The LWCCBuilder component can be dropped into the following type of Salesforce pages: App page, Home page, Record Page, Community Page.
 
-## Can LWCC components be used inside Aura Components?
-Yes, LWCC has been built using Lightning Web Components, so you can include them inside your Aura components.
+## Can LWCC components be used inside other Lighining Web Componentes?
```

Comment:
Typos ? Componentes => Components ?

Reply:
I was adding support for Spanish users... 😛